README.md

```diff
@@ -155,14 +155,14 @@ A good test is just completing a handshake.
 We supply the argument file, along with a corresponding test from `examples/tests/servers` as parameter.
 We get:
 
-    LD_LIBRARY_PATH=suts/openssl-1.1.1b/ java -jar target/dtls-fuzzer.jar @args/openssl/learn_openssl_server_psk -test examples/tests/servers/psk
+    LD_LIBRARY_PATH=suts/openssl-1.1.1b/ java -Dopenssl.version=1.1.1b -jar target/dtls-fuzzer.jar @args/openssl/learn_openssl_server_psk -test examples/tests/servers/psk
```

Comment:
Without specifying "-Dopenssl.version=1.1.1b", the default version "1.1.1k" will be used, which causes error.
![error](https://github.com/user-attachments/assets/c14f5371-7b75-4d56-9655-5c9756c02ab4)